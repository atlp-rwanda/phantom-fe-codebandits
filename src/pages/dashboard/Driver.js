import React from 'react';

function Driver() {
  return <h1 className="text-black">Drivers</h1>;
}

export default Driver;
