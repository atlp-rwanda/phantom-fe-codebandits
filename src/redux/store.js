import { configureStore } from '@reduxjs/toolkit';
import { authReducer } from './reducers/authReducer.js';
import { counterReducer } from './reducers/counterReducer.js';
import { updateProfileReducer } from './reducers/updateProfileReducer.js';
import { profileReducer } from './reducers/profileReducer.js';
import { managementReducer } from './reducers/managementReducer.js';
import { dashboardReducer } from './reducers/dashboardReducer.js';

export const store = configureStore({
  reducer: {
    counter: counterReducer,
    profileUpdates: updateProfileReducer,
    profileInfo: profileReducer,
    auth: authReducer,
    management: managementReducer,
    dashboard: dashboardReducer
  }
});
