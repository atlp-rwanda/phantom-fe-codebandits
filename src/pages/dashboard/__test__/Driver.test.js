import React from 'react';
import { Provider } from 'react-redux';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import { store } from '../../../redux/store.js';
import Driver from '../Driver.js';

describe('Driver Registration ', () => {
    it('Should render only the driver form', () => {
      const elem = renderer
        .create(
          <Provider store={store}>
            <BrowserRouter>
              <Driver />
            </BrowserRouter>
          </Provider>
        )
        .toJSON();
      expect(elem).toMatchSnapshot();
    });

});
