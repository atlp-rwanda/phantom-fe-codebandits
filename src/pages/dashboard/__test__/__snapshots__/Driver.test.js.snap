// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Driver Registration  Should render only the driver form 1`] = `
<h1
  className="text-black"
>
  Drivers
</h1>
`;
